src/Plate/utils.ts

```diff
@@ -100,14 +81,20 @@ export function ensureCoordinatesInRange<
     column: number;
   },
 >(coordinates: T): T & Coordinates {
-  if (!COORDINATES_ROWS.includes(coordinates.row as Coordinates['row'])) {
+  if (
+    !new CoordinateSystem96Well()
+      .rows()
+      .includes(coordinates.row as Coordinates['row'])
```

Comment:
Can `as` be avoided by defining `row: Coordinates['row']` in the generic type definition?